auto3dseg/algorithm_templates/dints/scripts/algo.py

```diff
@@ -74,8 +74,20 @@ def fill_template_config(self, data_stats_file, output_path, **kwargs):
                 for p_k, shape_k in zip(patch_size, max_shape)
             ]
 
+            try:
```

Comment:
What are the considerations to use a try-except here?

Reply:
`class_names` is an optional key for users.

Reply:
According to the doc-string in https://github.com/Project-MONAI/MONAI/blob/8e725aa2f3c50ffd1c0c45e8917d85f20c6637d0/monai/bundle/config_parser.py#L197, can we use the following:
```
if data_src_cfg.get("class_names") is None:
    hyper_parameters.update(...)
```